index.html

```diff
@@ -6,19 +6,32 @@
     <link rel="stylesheet" type="text/css" href="assets/css/toDo.css">
     <script type="text/javascript" src="assets/js/lib/jquery-3.1.1.min.js"></script>
     <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
You'll need to move the bootstrap CSS to be BEFORE the custom CSS or else the boot strap styles will be applied over custom ones.